src/providers/springer/utils.py

```diff
@@ -0,0 +1,36 @@
+import requests
+import structlog
+logger = structlog.get_logger()
+
+def get_page_content(url):
+    try:
+        response = requests.get(url)
+        response.raise_for_status()  # Raise an HTTPError if the response status code indicates an error
```

Comment:
same: I don't think we need a comment, the function raise_for_status() is quite informative